src/settings/App.tsx

```diff
@@ -266,6 +286,17 @@ export class App extends Component<{}, AppState> implements OC.Plugin<OC.Search.
 					onClick={() => {
 						this.setState({editingGroup: 0, editingMountPoint: 0})
 					}}>
+			<div id="groupfolders-admin-delegation">
+				<h3>{ t('groupfolders', 'groupfolders delegation') }</h3>
+				<em>{ t('groupfolders', 'Nextcloud allows you to delegate the administration of groupfolders to non-admin users.') }</em>
+				<br/>
+				<em>{ t('groupfolders', "Specify hereunder the groups that allowed to manage groupfolders and use its API's.") }</em>
+				<br/>
+			</div>
+			<GroupSelect
+                                allGroups={this.state.groups}
+				onChange={this.updateDelegatedAdminGroups}/>
```

Comment:
```suggestion
				onChange={() => this.updateDelegatedAdminGroups}/>
```

The function is not automatically bound to the component, see https://reactjs.org/docs/faq-functions.html#arrow-function-in-render for further reference

Reply:
I've used onChange={this.updateDelegatedAdminGroups.bind(this)} which is used all over the place in the app. It's said to be equivalent